index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Carter+One&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+    <title>Zellij</title>
+  </head>
+  <body>
+    <header class="interface">
+      <h1 class="game-title">Zellij</h1>
+
+      <form id="create-new-game" class="player-info">
+        <div class="player-name">
+          <label class="sr-only" for="nameone">Player 1:</label>
+          <input
+            class="player-name-input"
+            type="text"
+            id="nameone"
+            name="player_name"
+            placeholder="Player 1"
+          />
+        </div>
+        <div class="player-name">
+          <label class="sr-only" for="nametwo">Player 2:</label>
+          <input
+            class="player-name-input"
+            type="text"
+            id="nametwo"
+            name="player_name"
+            placeholder="Player 2"
+          />
+        </div>
+        <button class="start-two-players" type="submit">
+          <span>New Game</span>
+        </button>
+      </form>
+    </header>
+    <section class="game-wrapper">
+      <canvas class="game-surface" width="1280" height="720"></canvas>
+      <button class="help">
+        <img src="images/help.svg" width="35px" height="35px" alt="Help" />
+      </button>
+    </section>
+    <footer class="footer">
+      <small class="attribution">
+        <p>
```

Comment:
<p> tag can't be within <small>. The latter is inline element, and it can't wrap block-level one. [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Block-level_elements)